core/src/main/java/com/klaviyo/coresdk/networking/requests/IdentifyRequest.kt

```diff
@@ -1,13 +1,14 @@
-package com.klaviyo.coresdk.networking
+package com.klaviyo.coresdk.networking.requests
 
 import com.klaviyo.coresdk.KlaviyoConfig
+import com.klaviyo.coresdk.networking.RequestMethod
 import org.json.JSONObject
 
-class IdentifyRequest (
+internal class IdentifyRequest (
```

Comment:
There will be a few instances of this internal tag in this PR. Just internalizing some areas that I missed before to hide more of the SDK's implementation from the user.